frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -20,19 +23,41 @@ export const detectIndependentMode: ActionFeatureFlagDetector = (dispatch) =>
     },
   );
```

Comment:
It would be great if you could join all these together and perform all the flag-related operations in a single function.